app/code/Magento/InventoryImportExport/etc/di.xml

```diff
@@ -27,8 +27,10 @@
         </arguments>
     </type>
 
-    <!-- Source Item Import -->
-    <preference for="\Magento\CatalogImportExport\Model\StockItemImporterInterface" type="\Magento\InventoryImportExport\Model\StockItemImporter" />
+    <!-- Source Item Import Plugin -->
+    <type name="Magento\CatalogImportExport\Model\StockItemImporter">
```

Comment:
StockItemImporterInterface should be pluginized , not the precise implementation